### DISCUSSION TITLE
Creating an alternate home layout for posts

### DISCUSSION THREAD

**Discussion by imagelessthought** [cotes2020/jekyll-theme-chirpy#1331] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

I want to create an alternative version of the home page, which uses the home layout.  The home page lists the available posts. I copied the home layout exactly into home-alt.html and created a new markup file.  The posts are listed but the formatting is lost - the preview image is about 1rem by 1rem and a lot of the styling is missing.  If I point the md file to home.html it works fine, but when I change it to home-posts, even though it is exactly the same - it does not work.

Anyone know what I need to change?

alt-posts.md:
---
layout: home-posts
permalink: /posts-alt/
# Index page
---

Again, the layout of home-posts.html is a direct copy of the home.html layout. 

### Operations you have already tried

I tried creating seprate layouts and looked through all f the layouts - but could not find anything.  Lost of troubleshooting.

**Comment by Zeyun-Deng** [comment]
also curious for this problem...

**Comment by SoumyaK4** [comment]
@imagelessthought  This happened because you didn't create/apply necessary styles to the custom layout
